src/command-line/index.js

```diff
@@ -43,5 +43,5 @@ require("./edit");
 program.parse(argv.args);
 
 if (!program.args.length) {
-	program.parse(process.argv.concat("start"));
+	program.parse(process.argv.concat("--help"));
```

Comment:
I wasn't clear so I asked clarification to @xPaw and:

- `--help` is auto-generated by `commander`, not something we define, so we can't use `isDefault` on it.
- Replace `program.parse(process.argv.concat("--help"));` with `program.help()`.